lib/tran_sock.c

```diff
@@ -391,61 +391,61 @@ tran_sock_init(vfu_ctx_t *vfu_ctx)
 {
     struct sockaddr_un addr = { .sun_family = AF_UNIX };
     tran_sock_t *ts = NULL;
-    int ret;
+    int err;
 
     assert(vfu_ctx != NULL);
 
     ts = calloc(1, sizeof(tran_sock_t));
 
     if (ts == NULL) {
-        ret = -errno;
+        err = errno;
         goto out;
     }
 
     ts->listen_fd = -1;
     ts->conn_fd = -1;
 
     if ((ts->listen_fd = socket(AF_UNIX, SOCK_STREAM, 0)) == -1) {
-        ret = -errno;
+        err = errno;
         goto out;
     }
 
     if (vfu_ctx->flags & LIBVFIO_USER_FLAG_ATTACH_NB) {
-        ret = fcntl(ts->listen_fd, F_SETFL,
+        err = fcntl(ts->listen_fd, F_SETFL,
                     fcntl(ts->listen_fd, F_GETFL, 0) | O_NONBLOCK);
-        if (ret < 0) {
-            ret = -errno;
+        if (err < 0) {
+            err = errno;
             goto out;
         }
     }
 
-    ret = snprintf(addr.sun_path, sizeof(addr.sun_path), "%s", vfu_ctx->uuid);
-    if (ret >= (int)sizeof(addr.sun_path)) {
-        ret = -ENAMETOOLONG;
-    }
-    if (ret < 0) {
+    err = snprintf(addr.sun_path, sizeof(addr.sun_path), "%s", vfu_ctx->uuid);
+    if (err >= (int)sizeof(addr.sun_path)) {
+        err = ENAMETOOLONG;
+    } else if (err < 0) {
+        err = errno;
```

Comment:
According to the manpage `snprintf` doesn't set `errno`.

Reply:
good spot